blog/content/post/2023-10-22-liquid-cooling.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Upgrading your gears with liquid cooling
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2023-10-22
+archives: "2023"
+
+tags:
+  - tag 1
+  - tag 2
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Introduction
+
+A time comes when you want to upgrade your gears in your desktop, say a CPU.
+Either your use case changes or you have different needs, which force you to
+look for a different, more beefy CPU. In general CPU exchange is limited to
+swapping the CPU in the socket. But is that all?
+
+## Upgrading your cooling solution
+
+Depending on your setup, your old CPU may have behaved well with the current
+cooling you have. It was selected to satisfy the needs of your old CPU by
+providing enough heat dissipation. But a CPU upgrade  may cause your cooling
+solution to be insufficient, especially if the performance gap between the old
+and new CPU is significant. In this blog post I will show you a short story of
+upgrading from Intel Core i5-12600K to Intel Core i9-14900K and a standard
+fan+radiator solution to liquid cooling solution and what difficulties you may
+encounter during such journey.
```

Comment:
@miczyg1 if you upgrade on top of some standard products, which can be found in the 3mdeb shop, it would be worth mentioning otherwise, please mention more information about the spec e.g. chassis model.

Reply:
There is no word this is Dasharo-based hardware, which IMHO is very important; maybe even mention that in the abstract and title.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590R43-R46